docs/architecture/security/definitions.mdx

```diff
@@ -4,6 +4,85 @@ sidebar_position: 2
 
 # Definitions
 
+As a security product features can often be tweaked or adjusted in the name of "security". This is a
+very broad term, with unclear meaning, sometimes leading to constantly shifting goals. Further, when
+communicating to another engineer, or to product about an intended feature, its implementation and
+constraints, the communicating parties may have a different view of what they assume an attacker to
+be able to do, and what they want to achieve, possibly even not having anything they want to
+achieve, just the notion of "I want a secure product".
+
+A formal approach here is needed: to aid in communication and to also provide a clear set of goals
```

Comment:
The changes defines a shared "language" that can be used to communicate, but nothing as to how teams are expected to escalate concerns if they are doubtful about the agreed security risks.